easybuild/easyconfigs/f/fermi-lite/fermi-lite-20190320-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,44 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'fermi-lite'
+version = '20190320'
+local_commit = 'b499514'
+
+homepage = 'https://github.com/lh3/fermi-lite'
+description = """Standalone C library for assembling Illumina short reads in small regions."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+github_account = 'lh3'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = [
+    '%(name)s-20190320_avoid-bwa-namespace-conflict.patch',
+    '%(name)s-20190320_build-shared-lib.patch',
+]
+checksums = [
+    'ea1230e298b8e7193a996b5aceeff7fea44ef00852b0b87d4fcb504bdca0e712',  # b499514.tar.gz
+    # fermi-lite-20190320_avoid-bwa-namespace-conflict.patch
+    '893f7489283a85cb0dabf2da71f10fc4527fbfe2b110d8618293732fa74eabab',
+    '46978098bb439d4772f263375eb67621489bb48853c182d7aab9b9da8a41494c',  # fermi-lite-20190320_build-shared-lib.patch
+]
+
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [('zlib', '1.2.11')]
+
+files_to_copy = [
+    (['fml-asm'], 'bin'),
+    (['*.a', '*.so*'], 'lib'),
```

Comment:
```suggestion
    (['*.a', '*.%s* % SHLIB_EXT'], 'lib'),
```

Reply:
These are actually not quite equivalent - `'*.so*'` will copy both `libfml.so` and `libfml.so.1`, whereas `'*.%s*' % SHLIB_EXT` only copies `libfml.so`.

Reply:
But I included the * in at the end?

Reply:
wow, not sure how I missed that. I even wrote it in my response and it still didn't register